cutplace/interface.py

```diff
@@ -524,9 +523,28 @@ def as_sql_create_table(self, dialect='ansi'):
             if len(constraint) > 0:
                 constraints += constraint + ",\n"
 
+        constraints = constraints.rsplit(',', 1)[0]
+
         create_table += constraints
 
-        create_table += ");"
+        create_table += "\n);"
+
+        con = None
+
+        try:
```

Comment:
You should be able to use `contextlib.closing` and the `with` statement to avoid the rather messy None/try/finally.